engine/installation/linux/docker-ce/ubuntu.md

```diff
@@ -227,6 +227,10 @@ the repository.
     ```bash
     $ sudo docker run hello-world
     ```
+    or if you run on armhf:
```

Comment:
Please add a newline after line 229 and after line 230. Code blocks need to have blank lines around them.